modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/MnemonicParser.java

```diff
@@ -60,7 +60,7 @@
  * modifier key is pressed.
  * </ul>
  */
-public class TextBinding {
+public class MnemonicParser {
```

Comment:
As mentioned in the comments, I prefer this rename to be reverted. It could be done separately as cleanup.

Reply:
I created [JDK-8276206](https://bugs.openjdk.java.net/browse/JDK-8276206) to track this.